sql/synth_recipes.sql

```diff
@@ -3168,7 +3168,7 @@ INSERT INTO `synth_recipes` VALUES (49919,1,0,0,0,0,0,94,0,0,0,4100,4242,14372,0
 INSERT INTO `synth_recipes` VALUES (49920,1,0,0,0,0,0,70,0,0,0,4100,4242,13754,0,0,0,0,0,0,0,851,855,855,855,1,1,1,1,'Black Cotehardie (desynth)');  -- 2021.11.15 Based on synth.  Sites that show are in agreement.
 INSERT INTO `synth_recipes` VALUES (49921,1,0,0,0,0,0,8,0,0,0,4100,4242,12696,0,0,0,0,0,0,0,817,817,850,850,2,3,2,3,'Leather Gloves (desynth)');  -- 2021.11.15 Based on synth.  Sites that show are in agreement.
 INSERT INTO `synth_recipes` VALUES (49922,1,0,0,0,0,0,43,0,0,0,4100,4242,13200,0,0,0,0,0,0,0,817,851,851,851,1,1,2,2,'Waistbelt (desynth)');  -- 2021.11.15 Based on synth.  Sites that show are in agreement.
-INSERT INTO `synth_recipes` VALUES (49923,1,0,0,0,0,0,30,0,0,0,4100,4242,12826,0,0,0,0,0,0,0,817,817,817,817,6,6,6,6,'Studded Trousers (desynth)');
+INSERT INTO `synth_recipes` VALUES (49923,1,0,0,0,0,0,30,0,0,0,4100,4242,12826,0,0,0,0,0,0,0,817,817,651,848,6,6,2,1,'Studded Trousers (desynth)'); -- 2022.03.01 Basded on FFXIclopedia then change to reflect normal desynth behavior (says dhamlmel leather X 6 but recepie only use one)
```

Comment:
Studded Trousers
Lets stick with what XiClopedia sais unless we have direct evidence from retail.
Yield: [Grass Thread](https://ffxiclopedia.fandom.com/wiki/Grass_Thread) x 6
HQ 1: [Dhalmel Leather](https://ffxiclopedia.fandom.com/wiki/Dhalmel_Leather) x 6
HQ 2: [Iron Ingot](https://ffxiclopedia.fandom.com/wiki/Iron_Ingot) x 2